exporter/device.go

```diff
@@ -0,0 +1,38 @@
+package exporter
+
+import (
+	"context"
+	"time"
+
+	"github.com/ffddorf/unms-exporter/client/devices"
+	"github.com/ffddorf/unms-exporter/models"
+)
+
+type Device struct {
+	*models.DeviceStatusOverview
+}
```

Comment:
why did you make a new struct for this? i don't see what difference it would make if the `models. DeviceStatusOverview` type was used directly or just aliased instead of being embedded?

Reply:
This is in preparation for #23, where this will look like

```go
type Device struct {
	Statistics *models.DeviceStatistics
	*models.DeviceStatusOverview
}
```

Reply:
i'd prefer to not prematurely make this change. let's keep the PR small.